src/client/a2/main/grid/html/html-grid.component.pug

```diff
@@ -7,6 +7,7 @@
             [ngClass]="pulseClass(beat, pulse, pulses, pulseIndex)")
           .actual.note([ngClass]="noteClass(pulses)", *ngIf="player.value(key, pulseIndex)")
   .top-fx.overlay
+    pixi-effects
```

Comment:
Come to think of it, didn't we discuss using pixi in the back-fx layer instead?  I've already been experimenting with using html/css for top-fx.